README.md

```diff
@@ -1,4 +1,5 @@
 [![Build Status](https://travis-ci.org/ogier/pflag.png?branch=master)](https://travis-ci.org/ogier/pflag)
+[![GoDoc](https://godoc.org/github.com/spf13/pflag?status.svg)][3]
```

Comment:
Should probably point to the doc of this repo and not the fork. While you are at it, also replace the png from travis with the scaling svg version.